certificate-service/pdfbox-generator/src/main/java/se/inera/intyg/certificateservice/pdfboxgenerator/pdf/text/PdfTextGenerator.java

```diff
@@ -42,9 +42,10 @@ private void addText(
       Float offsetY,
       boolean isBold,
       int mcid,
-      PDStructureElement section
+      PDStructureElement section,
+      int pageIndex
   ) throws IOException {
-    final var page = pdf.getPage(0);
+    final var page = pdf.getPage(pageIndex);
```

Comment:
I get that. But I think in this instans it could be OK that it is specified for each certificate type. This can vary between different types and when it comes to other that FK certificates it could look very different.